config/development.json

```diff
@@ -4,7 +4,7 @@
 	"favicon_url": "/calypso/images/favicons/favicon-development.ico",
 	"client_slug": "browser",
 	"directly_rtm_widget_environment": "sandbox",
-	"protocol": "http",
+	"protocol": "https",
```

Comment:
This should be in the test instructions instead of the PR

Reply:
I agree, I was planning to revert this change.